### PR TITLE
Add Templates and labeler workflow

### PR DESCRIPTION
**Pull request by ByteOtter**
Add templates for issues and pull requests aswell as a workflow to label PRs.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+contact_links:
+  - name: ❓ Questions
+    url: https://github.com/cobbler/cobbler/discussions
+    about: Ask questions here
+  - name: 💬 Gitter Chat
+    url: https://gitter.im/cobbler/community
+    about: Chat with the Cobbler community
+  - name: 📬 Development mailing list
+    url: https://groups.google.com/forum/#!forum/cobbler-users
+    about: The developer mailing list
```
Comment (SchoolGuy) [inline .github/ISSUE_TEMPLATE/config.yml:11]: Let's try to not mention that please.

File: .github/ISSUE_TEMPLATE/03_backport_request.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+### Original feature issue
+
+<!--- (if present): Describe the feature --->
+<!--- Is there an PR number for that feature? --->
+
+### Target release
+
+- [ ] release33
+- [ ] release32
+- [ ] release30
```
Comment (SchoolGuy) [inline .github/ISSUE_TEMPLATE/03_backport_request.md:19]: That is not yet existing. I think we need to find a more suitable point for this.

Comment (ByteOtter) [reply .github/ISSUE_TEMPLATE/03_backport_request.md]: removed 👍🏼

File: .github/ISSUE_TEMPLATE/01_bug_report.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+1.
+2.
+3.
+
+### Expected behavior
+
+<!--- A clear and concise description of what you expected to happen. -->
+
+### Cobbler version
```
Comment (SchoolGuy) [inline .github/ISSUE_TEMPLATE/01_bug_report.md:26]: We want the version probably more then Cobbler itself but also Cobbler-TFTP

File: .github/ISSUE_TEMPLATE/01_bug_report.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+<!--- Paste output from `cobbler version` -->
+````paste below
+````
+
+### Operating system
+
+<!--- On which operating system do you use Cobbler? -->
+
+### Cobbler log
```
Comment (SchoolGuy) [inline .github/ISSUE_TEMPLATE/01_bug_report.md:36]: Same concern as before. We maybe also want the log of the application and not of the main daemon.

**Review by SchoolGuy — COMMENTED** [review]
Small additional remarks. :)

File: .github/ISSUE_TEMPLATE/01_bug_report.md (hunk 4 above)
Comment (ByteOtter) [reply .github/ISSUE_TEMPLATE/01_bug_report.md:36]: should a log be created in the same directory where cobbler creates its log or a seperate folder like `/var/log/cobblert-tftp/cobbler-tftp.log`? I guess the latter?

Comment (SchoolGuy) [reply .github/ISSUE_TEMPLATE/01_bug_report.md]: A separate folder is desired.

File: .github/ISSUE_TEMPLATE/01_bug_report.md (hunk 3 above)
Comment (ByteOtter) [reply .github/ISSUE_TEMPLATE/01_bug_report.md]: Fixed it already and made a note to get the cobbler version of any connected cobbler server when calling `cobbler-tftp version`

File: .github/ISSUE_TEMPLATE/01_bug_report.md (hunk 4 above)
Comment (ByteOtter) [reply .github/ISSUE_TEMPLATE/01_bug_report.md]: noted

**Comment by ByteOtter** [comment]
> @ByteOtter Since the Markdown linter complains on Codacy, we should either add one in a separate Pull Request that uses our opinionated style guide for Markdown or we should fixup as much as possible.

From what I understand, Codacy can't really handle the github specific things. The checkboxes and the title preset for PRs and Issues and such. Do we need a Markdown linter at all? If not we could probably tell Codacy to stop checking Markdown files?

**Review by SchoolGuy — APPROVED** [review]
LGTM. Thanks for the contribution!

**Pull request merged**